KarhooUISDK/Controllers/Payment/ThreeDSecure/AdyenThreeDSecureUtils.swift

```diff
@@ -11,7 +11,7 @@ public struct AdyenThreeDSecureUtils: ThreeDSecureUtils {
     public init() {}
     
     public var userAgent: String {
-        return "KH/UISDK/iOS/\(KarhooUISDKVersionNumber)"
+        "KH/UISDK/iOS/1.7.3" //\(KarhooUISDKVersionNumber)"
```

Comment:
We should look into fixing this instead of hardcoding the version number. Or at least update the release docs in Confluence to mention that this needs to be bumped for each release until it's fixed

Reply:
I've updated the internal release documentation to include this step until we fix it